modules/is_analytics_dashboard_master/manifests/params.pp

```diff
@@ -0,0 +1,100 @@
+# ----------------------------------------------------------------------------
+#  Copyright (c) 2019 WSO2, Inc. http://www.wso2.org
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ----------------------------------------------------------------------------
+
+# Claas is_analytics_dashboard::params
```

Comment:
Spellings mistake. *Class

Reply:
Resolved https://github.com/wso2/puppet-is/pull/60/commits/4db1eb3bd9b5f960a543b0b32012f23fbd0e2659